platforms/intellij/compose/gradle.properties

```diff
@@ -2,3 +2,4 @@ POM_ARTIFACT_ID=skate-compose
 POM_NAME=Skate (Compose)
 POM_DESCRIPTION=Skate (Compose)
 INTELLIJ_PLUGIN=true
+roborazzi.record.filePathStrategy=relativePathFromRoborazziContextOutputDirectory
```

Comment:
Can we add a comment for posterity on this?

Reply:
Sounds good, I added a comment `# Set the default path for Roborazzi record images` 992b80dd7906e3e8b7bf586d7afed8d3bf7cfe69